### PR TITLE
Implementation of authentication with IDP by using SAML

### PR DESCRIPTION
**Pull request by esvalukhin**
These changes are required for storing parameters of SAML identity providers in database which have been entered on UI. SAML providers' details are used for configuring identity providers during the startup of authorization server and during the procedure of getting information about supported SAML providers.

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package com.epam.ta.reportportal.database.entity.settings;
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java]: Could you please add the copyright header as in other classes?

File: src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+import java.io.Serializable;
+
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java]: Could you please add the author and some java docs describing the class?

File: src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java (hunk 1 above)
Comment (esvalukhin) [reply src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java]: Added licence to the file header

File: src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java (hunk 2 above)
Comment (esvalukhin) [reply src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java]: Javadoc for the parameters have been added

**Pull request merged**